contracts/contracts/interfaces/l1/ILineaRollup.sol

```diff
@@ -319,14 +269,14 @@ interface ILineaRollup {
   error BytesLengthNotMultipleOf32();
 
   /**
-   * @dev Thrown when the snarkhash is the zero hash.
+   * @dev Thrown when the computed shnarf does not match what is expected.
    */
-  error SnarkHashIsZeroHash();
+  error FinalShnarfWrong(bytes32 expected, bytes32 value);
 
   /**
-   * @dev Thrown when the computed shnarf does not match what is expected.
+   * @dev Thrown when a parent shnarf does not exist.
    */
-  error FinalShnarfWrong(bytes32 expected, bytes32 value);
+  error ParentShnarfDoesNotExist(bytes32 shnarf);
```

Comment:
```suggestion
  error ParentBlobNotSubmmitted(bytes32 shnarf);
```

I think this is more indicative of the underlying cause.

Reply:
addressed in new commit